ush/python/pygfs/task/land_analysis.py

```diff
@@ -0,0 +1,265 @@
+                'LAND_WINDOW_BEGIN': _window_begin,
+                'LAND_WINDOW_LENGTH': f"PT{self.config['assim_freq']}H",
+                'OPREFIX': f"{self.runtime_config.CDUMP}.t{self.runtime_config.cyc:02d}z.",  # TODO: CDUMP is being replaced by RUN
+                'APREFIX': f"{self.runtime_config.CDUMP}.t{self.runtime_config.cyc:02d}z.",  # TODO: CDUMP is being replaced by RUN
+                'GPREFIX': f"gdas.t{self.runtime_config.previous_cycle.hour:02d}z.",
+                'fv3jedi_yaml': _fv3jedi_yaml,
+            }
+        )
+
+        # task_config is everything that this task should need
+        self.task_config = AttrDict(**self.config, **self.runtime_config, **local_dict)
+
+    @logit(logger)
+    def initialize(self: Analysis) -> None:
+        """Initialize a global land analysis
+
+        This method will initialize a global land analysis using JEDI.
+        This includes:
+        #- staging CRTM fix files
+        - staging FV3-JEDI fix files
+        #- staging B error files
+        - staging model backgrounds
+        - generating a YAML file for the JEDI executable
+        - linking the JEDI executable (TODO make it copyable, requires JEDI fix)
+        - creating output directories
+        """
+        super().initialize()
+
+        # stage fix files
+        jedi_fix_list_path = os.path.join(self.task_config['HOMEgfs'], 'parm', 'parm_gdas', 'land_jedi_fix.yaml')
+        logger.debug(f"Staging JEDI fix files from {jedi_fix_list_path}")
+        jedi_fix_list = parse_yamltmpl(jedi_fix_list_path, self.task_config)
+        FileHandler(jedi_fix_list).sync()
+
+        # stage backgrounds
+        FileHandler(self.get_bkg_dict(AttrDict(self.task_config, **self.task_config))).sync()
+
+        # generate letkfoi YAML file
+        logger.debug(f"Generate variational YAML file: {self.task_config.fv3jedi_yaml}")
+        letkfoi_yaml = parse_j2yaml(self.task_config['LANDVARYAML'], self.task_config)
+        save_as_yaml(letkfoi_yaml, self.task_config.fv3jedi_yaml)
+        logger.info(f"Wrote letkfoi YAML to: {self.task_config.fv3jedi_yaml}")
+
+        # link executable to DATA/ directory
+        exe_src = self.task_config['JEDIEXE']
+        logger.debug(f"Link executable {exe_src} to DATA/")  # TODO: linking is not permitted per EE2.  Needs work in JEDI to be able to copy the exec.
+        exe_dest = os.path.join(self.task_config['DATA'], os.path.basename(exe_src))
+        if os.path.exists(exe_dest):
+            rm_p(exe_dest)
+        os.symlink(exe_src, exe_dest)
+
+        # need output dir for diags and anl
+        logger.debug("Create empty output [anl, diags] directories to receive output from executable")
+        newdirs = [
+            os.path.join(self.task_config['DATA'], 'anl'),
+            os.path.join(self.task_config['DATA'], 'diags'),
+        ]
+        FileHandler({'mkdir': newdirs}).sync()
+
+    @logit(logger)
+    def execute(self: Analysis) -> None:
```

Comment:
what is the use of this method if a shell script is being used?

Reply:
The `execute` function here is a place holder, which is not used in the land DA. I can delete this function.

Reply:
The `execute` function is removed.